docs/Advanced-Usage/Manager/Manager-Environment-Variables.rst

```diff
@@ -14,9 +14,10 @@ by the manager.
 This environment variable is used to prefix all Run Farm tags with some prefix in the AWS EC2 case.
 This is useful for separating run farms between multiple copies of FireSim.
 
-This is set in ``sourceme-f1-manager.sh``, so you can change it and commit it
-(e.g. if you're maintaining a branch for special runs). It can be unset or set
-to the empty string.
-
+.. _buildfarm-prefix:
 
+``FIRESIM_BUILDFARM_PREFIX``
+--------------------------
 
+This environment variable is used to prefix all Build Farm tags with some prefix in the AWS EC2 case.
+This is useful for separating build farms between multiple copies of FireSim.
```

Comment:
I'd keep the first sentence here, replace the second with "For CI use only."

Reply:
Ditto